### DISCUSSION TITLE
tags with values

### DISCUSSION THREAD

**Discussion by lev-immunai** [ckan/ckan#6584] [Q&A]
Can I add some value to tag, let's say I want to fing only datasets tagged with tag: "tag_name_XYZ"=123
and search by both tags and tags values later on
probably related to : https://github.com/ckan/ckan/discussions/6487

**Comment by frafra** [comment]
This is something you can already do by filtering by facet, as suggested in the link you proposed. There is also ckanext-scheming if you are interested in further customizing your schemas.

**Reply by lev-immunai** [reply]
thanks will try it
